### ISSUE TITLE
jasmine_node_test doesn't load files from 'test' folder

### ISSUE THREAD

**Issue by tqmukas** [bazelbuild/rules_nodejs#125]
I'm not sure if this is some kind of convention but it seems that `jasmine_node_test` rule simply doesn't load any files stored inside `test` folder.

My project configuration:
```bazel
# WORKSPACE

workspace(name = "app")

load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
git_repository(
    name = "build_bazel_rules_nodejs",
    remote = "https://github.com/bazelbuild/rules_nodejs.git",
    tag = "0.4.0",
)

load("@build_bazel_rules_nodejs//:defs.bzl", "node_repositories")
node_repositories(package_json = ["//:package.json"])
```

```bazel
# BUILD.bazel

package(default_visibility = ["//visibility:public"])

filegroup(
  name = "node_modules",
  srcs = glob(["node_modules/**/*"]),
)

jasmine_node_test(
  name = "test",
  srcs = ["test/helpers/babel.js"],
  node_modules = ":node_modules",
)
```

When running `bazel test :test` I get this warning message:

```
WARNING: /Users/.../app/BUILD.bazel:17:1: runfiles symlink test/helpers/babel.js -> /Users/.../app/test/helpers/babel.js obscured by test -> /private/var/tmp/_bazel_tomas/468a927e7e5460d6594ef02035f5311c/execroot/app/bazel-out/darwin-fastbuild/bin/test
```

and the `test.log`
```
exec ${PAGER:-/usr/bin/less} "$0" || exit 1
-----------------------------------------------------------------------------
failed to load main  Error: Cannot find module 'app/test/helpers/babel.js'
  looked in:
   app/test/helpers/babel.js,
   /private/var/tmp/_bazel_tomas/468a927e7e5460d6594ef02035f5311c/bazel-sandbox/3525228523712535370/execroot/app/bazel-out/darwin-fastbuild/bin/test.runfiles/app/test/helpers/babel.js,
   /private/var/tmp/_bazel_tomas/468a927e7e5460d6594ef02035f5311c/bazel-sandbox/3525228523712535370/execroot/app/bazel-out/darwin-fastbuild/bin/test.runfiles/app/node_modules/app/test/helpers/babel.js
    at Function.module.constructor._resolveFilename (/private/var/tmp/_bazel_tomas/468a927e7e5460d6594ef02035f5311c/execroot/app/bazel-out/darwin-fastbuild/bin/test_loader.js:152:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at /Users/tomas/_code/bazel-for-front-end/monorepo/app/node_modules/jasmine/lib/jasmine.js:88:5
    at Array.forEach (<anonymous>)
    at Jasmine.loadSpecs (/Users/.../app/node_modules/jasmine/lib/jasmine.js:87:18)
    at Jasmine.execute (/Users/.../app/node_modules/jasmine/lib/jasmine.js:206:8)
    at main (/private/var/tmp/_bazel_tomas/468a927e7e5460d6594ef02035f5311c/external/build_bazel_rules_nodejs/internal/jasmine_runner.js:47:11)
    at Object.<anonymous> (/private/var/tmp/_bazel_tomas/468a927e7e5460d6594ef02035f5311c/external/build_bazel_rules_nodejs/internal/jasmine_runner.js:52:22)
```

I would like to understand why this happens?

**Comment by alexeagle** [comment]
There's a collision in the output folder because your `jasmine_node_test` produces a file called `test` while you also have a  directory of the same name. Try giving your test a unique name.

**Comment by alexeagle** [comment]
Didn't hear back from you so I'll assume that last comment solved your problem.
